pandas/io/parsers.py

```diff
@@ -116,13 +116,16 @@
     Values to consider as True
 false_values : list, default None
     Values to consider as False
-skipinitialspace : boolean, default False
+skip_initial_space : boolean, default False
     Skip spaces after delimiter.
-skiprows : list-like or integer, default None
+skip_rows : list-like or integer, default None
     Line numbers to skip (0-indexed) or number of lines to skip (int)
     at the start of the file
-skipfooter : int, default 0
+skip_footer : int, default 0
     Number of lines at bottom of file to skip (Unsupported with engine='c')
```

Comment:
Can you clarify?  The reason why `skipfooter` is slightly different from the rest is because there were already duplicate args for these two, whereas the others don't.